changes/ticket34211

```diff
@@ -0,0 +1,3 @@
+  o Minor features (control signal in windows):
```

Comment:
The category we usually use here is just: `o Minor features (windows)`. Then it will be grouped together with the other windows changes.

Reply:
Thanks for the pointer, I have changed it!